base/util/src/main/java/com/netscape/cmsutil/crypto/CryptoUtil.java

```diff
@@ -172,6 +175,13 @@
 
     public static final int LINE_COUNT = 76;
 
+    private static SymmetricKey.Usage sess_key_usages[] = {
+        SymmetricKey.Usage.WRAP,
+        SymmetricKey.Usage.UNWRAP,
+        SymmetricKey.Usage.ENCRYPT,
```

Comment:
@jmagne What about leaving off ENCRYPT/DECRYPT from the session key usages? Just a thought...